vllm/core/block/block_table.py

```diff
@@ -65,6 +65,9 @@ def get_num_required_blocks(token_ids: List[int], block_size: int) -> int:
         """
         return cdiv(len(token_ids), block_size)
 
+    def get_blocks(self) -> Optional[List[Block]]:
+        return self._blocks
```

Comment:
nit: can use a property to reduce syntax overhead
```python3
@property
def blocks(self) -> Optional[List[Block]]:
    return self._blocks
```

then usage is simply `block_table.blocks` vs `block_table.get_blocks()`.